Utils/Dataflow/030_PDFAnalyzer/xmltable.py

```diff
@@ -133,7 +138,8 @@ def row_centery(row):
     return c
 
 class Table:
-    re_month = re.compile("(january|february|march|april|may|june|july|august|september|october|november|december)")
+    re_month = re.compile("(january|february|march|april|may|june|july|august\
+                          |september|october|november|december)")

```

Comment:
And please, check other regular expressions for both `\r` and newline+spaces within them.